vcluster/_fragments/patches.mdx

```diff
@@ -3,38 +3,71 @@ import CodeBlock from '@theme/CodeBlock';
+Patches override the default resource syncing rules in your vCluster YAML configurations.
 
-:::info Wildcard patches
-You can use `*` in paths to select all entries of an array or object, e.g. `spec.containers[*].name` or `spec.containers[*].volumeMounts[*]`. vCluster calls the patch multiple times when using the wildcard reference.
+By default, vCluster [syncs specific resources](https://www.vcluster.com/docs/vcluster/next/configure/vcluster-yaml/sync/) between virtual and host clusters. To modify the sync behavior, you can use patches to specify which fields to edit, exclude, or override during syncing.
+
+Patches can provide greater control over syncing by preventing unintended changes, preserving critical settings, and ensuring only necessary updates are applied.
+
+vCluster supports two patch types:
+
+- [JavaScript expression patch](#javascript-expression-patch): Uses JavaScript ES6 expressions to dynamically modify fields during syncing. You can define how a field changes when syncing from a virtual cluster to a host cluster, or from a host cluster to a virtual cluster.
+- [Reference patch](#reference-patch): Modifies specific fields within a resource to point to different resources. If the specified resource exists in a host cluster, vCluster automatically imports it into the corresponding virtual cluster.
+
+:::info Using wildcards in patches
+You can apply a wildcard, using an asterisk (`*`) in a specified path, to modify all elements of an array or object.
+For instance, `spec.containers[*].name` selects the `name` field of every container in the `spec.containers` array, and `spec.containers[*].volumeMounts[*]` selects all `volumeMounts` for each container. 
+When using the asterisk (`*`) notation, vCluster applies changes individually to every element that matches the path.
 :::
 
-### JavaScript Expression Patches
+### JavaScript expression patch
+
+A JavaScript expression patch allows you to use JavaScript ES6 expressions to change specific fields when syncing between virtual and host clusters. 
+This is useful when modifying resource configurations to align with differing environments or naming conventions between clusters.
+
+For example, if your virtual cluster follows a specific naming convention for container names, but the host cluster requires a different prefix, you can use a JavaScript expression patch to automatically update the relevant field for each container.
+
+You can define a path for `spec.containers[*].name` in `vcluster.yaml` using the following configuration:
 
-These are powerful JavaScript ES6 compatible expression patches that can be used to change a field while syncing. You define how it changes when syncing from the virtual cluster into the host cluster or when syncing from the host cluster into the virtual cluster. To change the path <span>{props.path}</span> you can do:
 <CodeBlock language="yaml">
-{`sync:
+{`
+sync:
   toHost:
-    ${props.resource}:
```

Comment:
@joaniekube we need to bring back this example into the code block so that the resources are dynamically updated since this fragment is used in multiple sections of the vcluster.yaml